packages/mobile/src/fiatExchanges/ProviderOptionsScreen.tsx

```diff
@@ -105,100 +97,81 @@ function ProviderOptionsScreen({ route, navigation }: Props) {
   const asyncUserLocation = useAsync(async () => fetchUserLocationData(countryCallingCode), [])
   const userLocation = asyncUserLocation.result
 
-  const asyncProviderQuotes = useAsync(async () => {
-    if (!account || !userLocation?.ipAddress || !isFocused) {
+  const asyncProviders = useAsync(async () => {
+    if (!userLocation) {
+      console.error(TAG, 'User location not yet set')
       return
     }
 
-    const simplexQuote = await fetchSimplexQuote(
-      account,
-      userLocation.ipAddress,
-      currencyToBuy,
-      localCurrency,
-      route.params.amount.crypto,
-      false
-    )
-
-    return { simplexQuote }
-  }, [userLocation?.ipAddress, isFocused])
-
-  const providerQuotes = asyncProviderQuotes.result
-
-  const {
-    MOONPAY_RESTRICTED,
-    SIMPLEX_RESTRICTED,
-    RAMP_RESTRICTED,
-    TRANSAK_RESTRICTED,
-    XANPOOL_RESTRICTED,
-  } = getProviderAvailability(userLocation)
-
-  const providerWidgetInputs = {
-    localAmount: route.params.amount.fiat,
-    currencyCode: localCurrency,
-    currencyToBuy,
-  }
-
-  const xanpool = {
-    id: CicoProviderNames.Xanpool,
-    paymentMethods: [PaymentMethod.Card, PaymentMethod.Bank],
-    restricted: XANPOOL_RESTRICTED,
-    onSelected: () => navigate(Screens.XanpoolScreen, providerWidgetInputs),
-  }
+    if (!isFocused) {
+      console.error(TAG, 'Screen is not in focus')
```

Comment:
Should we use `Logger.error` instead? Same for the other `console.error`s of this file

Reply:
I got a weird error when I used that. Left a comment to explain